### PR TITLE
Merge branch 'websites-preview' into preview [iss #5678]

### PR DESCRIPTION
**Pull request by vladimir-shcherbakov**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/Websites/Commands.Websites/Commands.Websites.Netcore.csproj [hunk 1]
```diff
@@ -39,7 +46,6 @@
   <ItemGroup>
     <ProjectReference Include="..\..\..\Common\Commands.Common.Authentication.Abstractions\Common.Authentication.Abstractions.Netcore.csproj" />
     <ProjectReference Include="..\..\..\Common\Commands.Common\Common.Netcore.csproj" />
-    <ProjectReference Include="..\..\Common\Commands.Common.Strategies\Common.Strategies.Netcore.csproj" />
```
Comment (markcowl) [inline src/ResourceManager/Websites/Commands.Websites/Commands.Websites.Netcore.csproj]: why is this removed?

File: src/ResourceManager/Websites/Commands.Websites/Commands.Websites.Netcore.csproj [hunk 2]
```diff
@@ -1,33 +1,40 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <Import Project="..\..\..\..\tools\Common.Netcore.Dependencies.targets" />
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <AssemblyName>Microsoft.Azure.Commands.Websites</AssemblyName>
+    <RuntimeIdentifiers>win10-x64</RuntimeIdentifiers>
```
Comment (markcowl) [inline src/ResourceManager/Websites/Commands.Websites/Commands.Websites.Netcore.csproj]: RuntimeIdentifier is unnecessary

**Comment by markcowl** [comment]
PR with fix is #6046

**Comment by vladimir-shcherbakov** [comment]
closed as a duplicate of the PR #6046
